test/integration/push_int_test.go

```diff
@@ -82,6 +82,7 @@ func (suite *PushIntegrationTestSuite) TestCarlisle() {
 	cp.Expect("(Y/n)")
 	cp.SendLine("y")
 	cp.Wait()
+	cp.ExpectExitCode(0)
```

Comment:
It makes sense to wait for the exit code here, but then the `cp.Wait()` needs to be removed.
```suggestion
	cp.ExpectExitCode(0)
```